src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripSettingsManagerTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Windows.Forms.Tests
+{
+    public class ToolStripSettingsManagerTests : IClassFixture<ThreadExceptionFixture>
+    {
+        [WinFormsFact]
+        public void Save_Load()
```

Comment:
We have a naming convention, we'll need to stick to it: https://github.com/dotnet/winforms/blob/master/docs/testing.md#naming

It should be something like:
```suggestion
        public void ToolStripSettingsManager_Save_Load_RoundTripExpected()
```